pkgs/top-level/ocaml-packages.nix

```
@@ -1054,6 +1052,8 @@ let
 
     ocamlnet = callPackage ../development/ocaml-modules/ocamlnet { };
 
+    ocaml_lwt = lwt;
```

```suggestion
    ocaml_lwt = lwt;
```
This is still out of order. As it is after `lwt`.